tests/test_xarray.py

```diff
@@ -180,7 +221,7 @@ def func(a, b):
 
 
 def test_strftime():
```

Comment:
Isn't this solely functionality in xarray now? If so, should probably just nuke the test since it's not covering anything within MetPy.

Reply:
Correct. I've just removed the test.